src/TypeCatalogGen/TypeCatalogGen.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Generates CorePsTypeCatalog.cs given powershell.inc</Description>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
We could use PowerShell.Common.props.

Reply:
Currently this is a separate project.  I think if we want all the csproj to depend on PS.Common.Props, let's do that as a separate PR.